node/test/graphql/query/config.spec.ts

```diff
@@ -91,23 +84,16 @@ describe('config', () => {
         }`,
       });
     expect(response.status).to.equal(200);
-    expect(response.body).to.eql({
-      data: {
-        config: {
-          cmi5Enabled: false,
-          cmi5Endpoint: '',
-          cmi5Fetch: '',
-          googleClientId: '',
-          mentorsDefault: [],
-          urlClassifier: '/classifier',
-          urlGraphql: '/graphql',
-          urlVideo: '/video',
-          styleHeaderLogo: '',
-        },
-      },
-      extensions: {
-        newToken: '',
-      },
+    expect(response.body.data.config).to.eql({
```

Comment:
make sure to use optional chaining here...if this test fails with data=null we want it to fail as an expectation mismatch and NOT a null-pointer ref that throws from the test code